<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prodManager</class>
 <widget class="QMainWindow" name="prodManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProdManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="qfLeftZone">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="vlLeftZone">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="hfPreview">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabManager">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabProject">
        <attribute name="title">
         <string>Project</string>
        </attribute>
        <layout class="QGridLayout" name="glTabProject">
         <property name="margin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="tabShots">
        <attribute name="title">
         <string>Shots</string>
        </attribute>
        <layout class="QGridLayout" name="glTabShots">
         <property name="margin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="tabLineTest">
        <attribute name="title">
         <string>LineTest</string>
        </attribute>
        <layout class="QGridLayout" name="glTabLineTest">
         <property name="margin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPref">
    <property name="title">
     <string>Pref</string>
    </property>
    <widget class="QMenu" name="menuStyle">
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="miDefaultStyle"/>
     <addaction name="miDarkGreyStyle"/>
     <addaction name="miDarkOrangeStyle"/>
    </widget>
    <addaction name="menuStyle"/>
   </widget>
   <addaction name="menuPref"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="miDefaultStyle">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="miDarkGreyStyle">
   <property name="text">
    <string>DarkGrey</string>
   </property>
  </action>
  <action name="miDarkOrangeStyle">
   <property name="text">
    <string>DarkOrange</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
